assessment/src/config/env.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Represents the configuration for environment variables.
+ */
+export class EnvConfig {
+	public static PORT = "PORT";
+	public static ENC_KEY = "ENC_KEY";
+
+	private static vars: Map<string, string>;
+
+	/**
+	 * Initializes the environment configuration.
+	 */
+	public static init() {
+		this.vars = new Map<string, string>();
+		this.loadEnv();
+	}
+
+	/**
+	 * Loads the environment variables.
+	 */
+	private static loadEnv() {
+		this.vars.set("PORT", process.env.PORT || "3000");
```

Comment:
Optional Nit: Use the variable instead of hardcoded string

Reply:
I would keep with string since if beginners google this will look closer to what they will find. We want to challenge them, but not completely overwhelm them with best practices.